### DISCUSSION TITLE
Frigate devices/entites with Frigate NVR Proxy for Frigate Hass Card in HA

### DISCUSSION THREAD

**Discussion by clangren** [blakeblackshear/frigate#3810] [Ask A Question] [answered]
Looking for help to setup camera devices and entities in HA so I can use the durmotduffy/frigate-hass-card lovelace card to its full potential.

The Frigate NVR Proxy AddOn is successfully running and presenting all functionality in HA, but I have no devices/entities to drive the frigate-hass-card setup. I tried the Generic Camera integration but it does not support all the Frigate-added functionality re: events, clips and such.

Overall environment is running in Proxmox 7 with setup as follows:
   - running HA as a VM under HassIO
   - running Frigate as a container in a LXC-Docker-Frigate setup
(debian-11-standard LXC container; docker compose; blakeblackshear/frigate:stable-amd64)

All aspects of the environment are updated to versions current through the date of this post.

Separated installs of Frigate and HA as recommended to facilitate larger storage for recordings and events.

Apologies if I missed finding this information in other posts. I have spent a couple of hours trying to sort this out and any input would be most appreciated (especially if the input is that just doesn’t work).

I posted this question on the HA community and will repost answers/feedback.

Thanks!

**Comment by NickM-27** [comment, marked as answer]
You need to install the integration https://github.com/blakeblackshear/frigate-hass-integration

**Reply by clangren** [reply]
Thank you very much for the reply and direction. I had the integration installed but did so before setup of Frigate in my LXC-Docker-Frigate setup. If my understanding is (now) correct, since the config.yml for Frigate was not present for the HA/Frigate integration to read, there was no way for the integration to know what to setup in HA. In the future if I add/change my config.yml in LXC-Docker-Frigate I will know to go rerun the integration so the HA/Frigate integration is aware of this as well.

Again, thanks very much!

**Reply by NickM-27** [reply]
That's not really correct, the integration does not read the config file at all, it just communicates with Frigate via MQTT & REST.

**Reply by clangren** [reply]
Thanks for the additional feedback and clarification. I must surely have messed it up in some fashion but it is working perfectly now.  I will correct the info I left on HA community as well.
